core/include/detray/materials/material_slab.hpp

```diff
@@ -0,0 +1,47 @@
+/** Detray library, part of the ACTS project (R&D line)
+ *
+ * (c) 2022 CERN for the benefit of the ACTS project
+ *
+ * Mozilla Public License Version 2.0
+ */
+
+#pragma once
+
+// Detray include(s)
+#include "detray/definitions/qualifiers.hpp"
+
+namespace detray {
+
+// Slab structure to be mapped on the mask
```

Comment:
For the energy loss formulate, one also needs often an access to the electron density, I suppose you plan to do this like:
```c++
slab.material().A()/slab.material().molar_electron_density()`
```
right?

Reply:
We can leave it as such, if - for convenience - we will introduce another member function, we can always do that.